src/comgr.cpp

```diff
@@ -585,14 +585,14 @@ class Dataset : ComgrOwner
     ~Dataset() { EC(amd_comgr_destroy_data_set(handle)); }
     auto GetHandle() const { return handle; }
     void AddData(const Data& d) const { EC_THROW(amd_comgr_data_set_add(handle, d.GetHandle())); }
-    void AddData(const std::string& name,
-                 const std::string& content,
+    void AddData(const fs::path& name,
```

Comment:
[Q] Why this change is necessary? Formally speaking, COMgr has nothing to do with `fs:path` semantics. Is it designed to abstract us from compiler and filesystem. `amd_comgr_set_data_name` deals with C strings that work as "text labels".

Reply:
[resolved][I do not care]